lib/client-assets.php

```diff
@@ -335,12 +335,13 @@ function gutenberg_register_packages_styles( $styles ) {
 	);
 	$styles->add_data( 'wp-components', 'rtl', 'replace' );
 
+	$block_library_filename = current_theme_supports( 'split-block-styles' ) ? 'common' : 'style';
```

Comment:
So we're building the block library styles in two ways: separately and together in order to support both themes that do or do not support splitting block styles right?

Reply:
Right. For backwards-compatibility, we need the combined stylesheet, so we need both methods.
Of course I could just change the URL of the combined stylesheet, call a PHP file instead and include (via PHP) the separate files in there. But it would probably be overkill...